README.md

```diff
@@ -344,3 +366,13 @@ like to apply them in Photoshop or any other code.
 | NeutralGrey           | 224 | 224 | 224 | \#E0E0E0 |
 | LightOchre            | 255 | 173 | 140 | \#FFAD8C |
 | UpdatedOchre          | 231 |  71 |  38 | \#E74726 |
```

Comment:
Are UpdatedOchre and Ochre same same but different?

Reply:
Would It be worth consolidating the duplicate colors into one?

Reply:
<img width="485" alt="image" src="https://github.com/user-attachments/assets/1e102129-c22b-41e2-aa39-ed9ebbf2eeca">

Yes, consolidating would be a worthy goal.

Reply:
It's like going to the nail salon

<img width="226" alt="Screenshot 2024-08-28 at 9 37 40 AM" src="https://github.com/user-attachments/assets/f82a907c-7156-4436-9f3d-fc8ebb3de874">